dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClasspathCheckOption.java

```diff
@@ -84,30 +85,35 @@ private static Options configureOptions() {
     options.addOption(
         "b", "bom", true, "BOM to generate a class path, specified by its Maven coordinates");
 
-    Option artifactOption = Option.builder("a").longOpt("artifacts").hasArgs()
-        .valueSeparator(',')
-        .desc("Maven coordinates for Maven artifacts (separated by ',') to generate a class path")
-        .build();
+    Option artifactOption =
+        Option.builder("a")
+            .longOpt("artifacts")
+            .hasArgs()
+            .valueSeparator(',')
+            .desc(
+                "Maven coordinates for Maven artifacts (separated by ',') to generate a class path")
+            .build();
     options.addOption(artifactOption);
 
-    Option jarOption = Option.builder("j").longOpt("jars").hasArgs()
-        .valueSeparator(',')
-        .desc("Jar files (separated by ',') to generate a class path")
-        .build();
+    Option jarOption =
+        Option.builder("j")
+            .longOpt("jars")
+            .hasArgs()
+            .valueSeparator(',')
+            .desc("Jar files (separated by ',') to generate a class path")
+            .build();
     options.addOption(jarOption);
 
-    options.addOption(
-        "r",
-        "report-only-reachable",
```

Comment:
This option has not been implemented, and being removed.

Reply:
we need this though. Per yesterday's discussion we want reachability checking to be configurable.

Reply:
This change is to mark errors unreachable if there is no paths to the class, **without dropping** errors based on the value.

Do you still want configuration?

Reply:
This should be OK for now then